### PR TITLE
[PRED-1593] Remove the -e from the pip installs for the package

### PR DESCRIPTION
**Pull request by smillet15**
Using an editable install was hiding a bug when the package was
refactored but the new sub-library was not being properly shipped.

### CODE REVIEW

**Comment by smillet15** [comment]
So, we have evidence that the tests fail as they should. 

https://travis-ci.org/datarobot/batch-scoring/builds/264599687?utm_source=github_status&utm_medium=notification
https://travis-ci.org/datarobot/batch-scoring/builds/264599611?utm_source=github_status&utm_medium=notification
https://ci.appveyor.com/project/Axik/batch-scoring/build/960
https://ci.appveyor.com/project/Axik/batch-scoring/build/961

Will rebase and I expect green with the PR that was just merged to master.

File: Dockerfile-3.5 [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN pip install -r requirements.txt -r requirements-test.txt
 
 COPY . /opt/project
 
-RUN pip install -e .
```
Comment (sevikkk) [inline Dockerfile-3.5]: These dockerfiles are used for in-place debug, and have volumes configured in docker-compose files, so removing -e from them will break running from docker-compose.

The same is for docker-entrypoint file.

So I don't have any objections to Makefile change and Appveyor config, but I don't think changes to docker related things are really required

Comment (smillet15) [inline Dockerfile-3.5]: @sevikkk Makes sense. I was just casting a wide net to get something up and confirm the failure and then fix. Will clean that up shortly.

**Comment by dsakagi** [comment]
So then... we don't want to do this?

**Comment by smillet15** [comment]
@dsakagi We want the appveyor and Makefile changes still since they will actually fix the underlying bug and make sure the package installs correctly for future changes.

**Comment by smillet15** [comment]
And on travis removing the `-e` breaks coverage? Hmmm.

**Comment by smillet15** [comment]
@sevikkk Can you take a look again when you have a moment. Seems like the appveyor tests are the only place we can drop the `-e` but that gives me enough confidence in catching wholesale packaging errors in the future.

**Pull request merged**